src/cloud-cli/cli.ts

```diff
@@ -55,34 +49,68 @@ program
     }
   });
 
-program
+/*** APPLICATIONS MANAGEMENT ***/
+const applicationCommands = program
+  .command('applications')
+  .description('Manage your DBOS applications')
+  .option('-h, --host <string>', 'Specify the host', DEFAULT_HOST)
+  .option('-p, --port <port>', 'Specify the port', DEFAULT_PORT)
```

Comment:
I don't think <port> means anything in this context. It's not a type (it is processed as a string by default) and there is already a description for the flag. If we agree I'd like to remove all of them.